docs/source/deployment/databricks.md

```diff
@@ -259,3 +259,7 @@ Kedro-Viz can then be launched in a new browser tab with the `%run_viz` line mag
 ```ipython
 In [2]: %run_viz
 ```
+
+### How to use Datasets stored on Databricks DBFS
+
+DBFS is a distributed file system mounted into a DataBricks workspace and accessible on a DataBricks cluster. It maps cloud object storage URIs to relative paths simplifying the process of persisting files. This is useful for libraries to read/write distributed storage as if it's a local file. Some of our datasets like `SparkDataSet` expect the filepath in DBFS to be prefixed with `/dbfs` this is because DataBricks will resolve any filepath given to PySpark in DBFS. However, when loading datasets using Pandas, DataBricks will resolve the filepath in the block storage volume attached to the driver node. This will result in a `DataSetError`, as the code will try to load files from the DBFS save location which doesn't exist in the local filesystem.
```

Comment:
```suggestion
DBFS is a distributed file system mounted into a DataBricks workspace and accessible on a DataBricks cluster. It maps cloud object storage URIs to relative paths so as to simplify the process of persisting files. With DBFS, libraries can read from or write to distributed storage as if it's a local file. 

Some of Kedro's datasets like `SparkDataSet` expect the file path in DBFS to be prefixed with `/dbfs` because DataBricks resolves any path given to PySpark in DBFS. **However, Kedro's Pandas datasets do not work in the same way** and will result in a `DataSetError`, as the code will try to load files from a DBFS save location which doesn't exist in the local filesystem.
```

Reply:
@SajidAlamQB I've made a few edits to simplify the text a bit as I don't think the reader needs all the information about the driver node/block storage volume for use with a pandas dataset. But feel free to revert/ignore my suggestion if you think they do need that info.

Reply:
Also, I didn't really feel like this was sufficient because you've titled the section "How to use..." but all it contains is a warning about an error. Don't you need to show how to use the datasets stored on DBFS so you don't get the error? That is, either present the code needed or a warning to say that you just cannot use Pandas because of the fact it's local.

Reply:
Good point maybe making this a `note` would be better? The main point we wanted to get across was that `SparkDataSet` expects file paths prefixed with `/dbfs`.

Reply:
Oh, that's interesting because that didn't come across at all to me, sorry. I would definitely revise then...Let me suggest again...